packages/@okta/i18n/src/properties/login.properties

```diff
@@ -448,6 +448,16 @@ verify.webauthn.biometric.instructions=Your browser or device will prompt you to
 webauthn.biometric.error.factorNotSupported = Security key or biometric authenticator is not supported on this browser. Select another factor or contact your admin for assistance.
 webauthn.biometric.error.factorNotSupported.oneFactor = Security key or biometric authenticator is not supported on this browser. Contact your admin for assistance.
 enroll.webauthn.instructions.noSupportForBiometric = Note: Some browsers may not support biometric authenticators.
+authfactor.webauthn.error.clientdata_parsing_failed=Client data parsing failed
+authfactor.webauthn.error.clientdata_message_type_mismatch=Message type in client data doesn't match expected message type.
+authfactor.webauthn.error.clientdata_challenge_missing=No challenge parameter in client data
+authfactor.webauthn.error.clientdata_challenge_mismatch=Challenge in client data doesn't match with server side value
+authfactor.webauthn.error.clientdata_origin_missing=No origin parameter in client data
+authfactor.webauthn.error.clientdata_origin_mismatch=Origin parameter in client data doesn't match. clientData origin was {0}
+authfactor.webauthn.error.invalid_enrollment_request_data=Invalid data in the WebAuthn enrollment request. {0}
+authfactor.webauthn.error.assertion_validation_failure=None of the WebAuthn authenticators the user has enrolled for was able to validate the assertion
+authfactor.webauthn.error.authdata_validation_failure_during_authentication_challenge=Failed to validate authenticator data during WebAuthn verification.
+authfactor.webauthn.error.signature_verification_failure=Signature verification was unsuccessful.
```

Comment:
@mpeng-okta I know these are api error messages. Now seems like a good time to make sure they are user friendly too.  @vincentvoong-okta  thoughts ?

Reply:
Yes.. We should override these messages here, which make sense to Enduser.